addons/web/static/src/legacy/scss/bootstrap_overridden.scss

```diff
@@ -53,6 +53,8 @@ $min-contrast-ratio: 3 !default;
 //
 // Quickly modify global styling by enabling or disabling optional features.
 $enable-smooth-scroll: false !default;
+$enable-grid-classes: true;
+$enable-cssgrid: true;
```

Comment:
It's very unlikely that people want to disable it, but it's better to add `!default` ;)